src/libraries/System.Private.CoreLib/src/System/Byte.cs

```diff
@@ -272,5 +279,704 @@ object IConvertible.ToType(Type type, IFormatProvider? provider)
         {
             return Convert.DefaultToType((IConvertible)this, type, provider);
         }
+
+#if FEATURE_GENERIC_MATH
+        //
+        // IAdditionOperators
+        //
+
+        [RequiresPreviewFeatures]
```

Comment:
Annotating the interface itself as preview isn't sufficient?  We also need to annotate all explicit implementations of that interface? cc: @pgovind

Reply:
Missed this notification in my email. We don't need to. We can also just suppress the analyzer here.

Reply:
> We don't need to. We can also just suppress the analyzer here.

Does this mean the analyzer currently requires it but you agree it shouldn't?

Reply:
> Does this mean the analyzer currently requires it

Yup. The analyzer does single level lookups right now and tags methods/properties etc when it find the attribute on them (or the class/interface). It works well for implicit implementations. For explicit implementations I need to think about it some more. If we had a 3rd party library author implement `IAdditionOperators` explicitly, I _think_ having a diagnostic at the implementation is good. That way the library author knows that they are implementing a preview interface. I'm hoping to get https://github.com/dotnet/roslyn-analyzers/pull/5155 in soon though and we can iterate on the design more concretely once we start consuming it in the runtime.